app/assets/stylesheets/shop.scss

```diff
@@ -376,7 +376,7 @@ a > h3 {
 
   h4 {
     display: inline;
-    font-size: 23px;
+    font-size: $title-text;
```

Comment:
This variable name should be more intuitive as we would be using it across the app, something like `gm-font-m` and 17px can be `gm-font-sm` etc